packages/calling/src/Events/types.ts

```diff
@@ -372,3 +374,16 @@ export type SanitizedEndTimeAndSessionId = {
   endTime: number;
   sessionId: string;
 };
+export type UCMLine = {
```

Comment:
Please leave a line between each of these new types. Please do the same for the older types above also. Looks like this was missed from earlier.

Reply:
Taken care of it.